samples/net/http_server/Kconfig

```diff
@@ -0,0 +1,49 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menu "HTTP server sample"
+
+config HTTP_SERVER_SAMPLE_SERVER_PORT
+	int "Server Port"
+	default 8080
+
+config HTTP_SERVER_SAMPLE_SERVER_PORT_TLS
+	int "Server Port (TLS)"
+	default 443
+
+config HTTP_SERVER_SAMPLE_CLIENTS_MAX
+	int "Maximum clients"
+	default 2
+	help
+	  This option sets the maximum number of concurrent clients.
```

Comment:
Might be interesting to add a sentence on the impact of increasing the number of clients (e..g memory footprint, performance, etc.)